fe/fe-core/src/main/java/com/starrocks/task/PublishVersionTask.java

```diff
@@ -107,11 +128,47 @@ public Set<Long> collectErrorReplicas() {
                 if (replica != null) {
                     errorReplicas.add(replica.getId());
                 } else {
-                    LOG.info("could not find related replica with tabletid={}, backendid={}",
-                            tabletId, this.getBackendId());
+                    LOG.info("could not find related replica with tabletid={}, backendid={}", tabletId, this.getBackendId());
                 }
             }
         }
         return errorReplicas;
     }
+
+    public void updateReplicaVersions(List<TTabletVersionPair> tabletVersions) {
+        if (span != null) {
+            span.addEvent("update_replica_version_start");
+            span.setAttribute("num_replicas", tabletVersions.size());
+        }
+        TabletInvertedIndex tablets = GlobalStateMgr.getCurrentInvertedIndex();
+        List<Long> tabletIds = tabletVersions.stream().map(tv -> tv.tablet_id).collect(Collectors.toList());
+        List<Replica> replicas = tablets.getReplicasOnBackendByTabletIds(tabletIds, backendId);
+        if (replicas == null) {
+            LOG.warn("backend not found backendid={}", backendId);
```

Comment:
```suggestion
            LOG.warn("replicas nout found on backend backendid={}", backendId);
```

Reply:
I read the code again, and found out from the comments:
```
    /**
     * For each tabletId in the tablet_id list, get the replica on specified backend or null, return as a list.
     *
     * @param tabletIds tablet_id list
     * @param backendId backendid
     * @return list of replica or null if backend not found
     */
    public List<Replica> getReplicasOnBackendByTabletIds(List<Long> tabletIds, long backendId) {
```

Actually, `null` means `backend not found` indeed, not `replicas not found`

Reply:
yes, you're right, i also read the code again, it means the specified backend not found.